Source/WTF/wtf/ProcessID.h

```diff
@@ -52,7 +52,7 @@ inline ProcessID getCurrentProcessID()
 {
 #if OS(WINDOWS)
     return GetCurrentProcessId();
-#elif OS(MORPHOS) || OS(AMIGAOS)
+#elif OS(MORPHOS)
 	return FindTask(0)->tc_ETask->UniqueID;
 #else
     return getpid();
```

Comment:
presumably, under the hood, the C-library performs something similar to FindTask(NULL) and returns the ID when using "getpid()"?

Reply:
Yeah, I think so.